conf/machine/hikey-32.conf

```diff
@@ -0,0 +1,45 @@
+#@TYPE: Machine
+#@NAME: LeMaker HiKey machine
+#@DESCRIPTION: LeMaker HiKey machine
+
+require conf/machine/include/tune-cortexa8.inc
+DEFAULTTUNE = "cortexa8hf-neon"
```

Comment:
Ok, I think I got the point. I will move this to the DISTRO layer.  I would suggest that the Mali related cleanup should be targeted separately, since it is not the goal of this patch.